src/cljs/main/broadfcui/page/library/library_page.cljs

```diff
@@ -311,6 +318,5 @@
                          (select-keys @state [:library-attributes :search-result-columns :search-text
                                               :facet-filters :aggregate-fields :expanded-aggregates]))])]])
    :component-did-update
-   (fn [{:keys [state refs]}]
-     (persistence/save {:key PERSISTENCE-KEY :state state})
-     (react/call :execute-search (@refs "dataset-table")))})
+   (fn [{:keys [state]}]
+     (persistence/save {:key PERSISTENCE-KEY :state state :except [:library-attributes]}))})
```

Comment:
This was the bulk of the problem when reloading.  Don't re-search on every update, explicitly do it when changing the search text or facets.